src/panels/lovelace/components/notifications/hui-notifications-button.ts

```diff
@@ -0,0 +1,57 @@
+import {
+  html,
+  LitElement,
+  TemplateResult,
+  css,
+  CSSResult,
+  property,
+} from "lit-element";
+import "@polymer/paper-badge/paper-badge";
+import "@polymer/paper-icon-button/paper-icon-button";
+import { fireEvent } from "../../../../common/dom/fire_event";
+
+class HuiNotificationsButton extends LitElement {
+  @property() public notifications?: string[];
+  @property() public open?: boolean;
+
+  protected render(): TemplateResult | void {
+    return html`
+      <paper-icon-button
+        icon="hass:bell"
+        @click="${this._clicked}"
+      ></paper-icon-button>
+      ${this.notifications
+        ? html`
+            <paper-badge label="${this.notifications.length}"></paper-badge>
+          `
+        : ""}
+    `;
+  }
+
+  static get styles(): CSSResult[] {
+    return [
+      css`
+        :host {
+          position: relative;
+        }
+        paper-badge {
+          left: 23px !important;
+          top: 0px !important;
+        }
+      `,
+    ];
+  }
+
+  private _clicked() {
+    this.open = true;
+    fireEvent(this, "open-changed");
```

Comment:
you should include the value as event detail `{ value: this.open }`

Reply:
Reason you have to manually fire these events is because Lit doesn't have the notion of `notify: true` as Polymer had.